packages/rum-core/src/error-logging/error-logging.js

```diff
@@ -211,6 +203,32 @@ class ErrorLogging {
     }
     return this.logErrorEvent(errorEvent)
   }
+
+  _parseRejectReason(reason) {
+    const errorEvent = {
+      message: PROMISE_REJECTION_PREFIX
+    }
+
+    if (Array.isArray(reason)) {
+      errorEvent.message += '<object>'
+    } else if (typeof reason === 'object') {
+      try {
+        errorEvent.message += JSON.stringify(reason)
+        errorEvent.error = reason
+      } catch (error) {
+        // fallback. JSON.stringify can throw exceptions in different circumstances.
+        // please, see this link for more info:
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#exceptions
+        errorEvent.message += '<object>'
+      }
+    } else if (typeof reason === 'function') {
+      errorEvent.message += '<function>'
+    } else {
+      errorEvent.message += reason
```

Comment:
Did we handle this case before?

Reply:
yes, this case was:

https://github.com/elastic/apm-agent-rum-js/blob/main/packages/rum-core/src/error-logging/error-logging.js#L197